myhpom/static/myhpom/include/_upload_widget.scss

```diff
@@ -0,0 +1,52 @@
+.advance-directive-widget {
+    background-color: white;
+    border: 1px solid black;
+}
+
+.advance-directive-widget__breadcrumbs {
+    border-bottom: 1px solid black;
+    margin-bottom: 0;
+    padding: 1rem;
+    list-style: none;
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
+
+    @include media-breakpoint-down(md) {
+        flex-direction: column;
+        align-items: flex-start;
+    }
+
+    @include media-breakpoint-down(sm) {
+        flex-direction: row;
+        align-items: center;
+    }
+}
+
+.advance-directive-widget__breadcrumb-item {
+    display: inline-flex;
+    flex-wrap: nowrap;
+    align-items: center;
+    font-size: 80%;
+
+    &:not(:first-child)::before {
+        font-family: 'Icons';
+        font-size: 80%;
+        line-height: 75%;
+        content:'\e036';
+        display: inline-block;
+        margin: 0 0.35rem;
+    }
+
+    @include media-breakpoint-down(md) {
+        font-size: 100%;
+    }
+
+    @include media-breakpoint-up(xl) {
+        font-size: 100%;
+    }
+}
+
+.advance-directive-widget__body {
+    padding: 2rem;
```

Comment:
This does look a bit narrow between the `md`/`lg` breakpoints, and the wireframe suggest that the breadcrumbs have the same padding as the body...I like this additional padding at the other breakpoitns though...I think this all might look better if the dashboard allocated more space to the 'ad widget' in these breakpoints (and you could probably get rid of the `flex-direction: column` as well).